src/v2/Apps/Order/Components/AddressModal.tsx

```diff
@@ -72,6 +75,14 @@ export const AddressModal: React.FC<Props> = ({
   modalDetails,
   me,
 }) => {
+  const [countryCode, setCountryCode] = useState<string>("us")
```

Comment:
non-blocking - can we use a [user's location](https://github.com/artsy/metaphysics/blob/master/_schema.graphql#L15028-L15040) as the default country?

Reply:
Yes, that's been discussed [here](https://artsy.slack.com/archives/C2TQ4PT8R/p1629920350084200) and also [here](https://artsy.slack.com/archives/C01ADJNCS5D/p1628605963138300), but I asked Charles if this geolocating should be done on our first pass and the answer was no (for now).